editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/adopters/configurators/DefaultTemplateConfigurator.java

```diff
@@ -61,13 +61,8 @@ public boolean configure(IProject project, CommonNewProjectMetaData metadata, IP
 		IProjectFacetVersion javaFacet = ProjectFacetsManager.getProjectFacet("jst.java").getDefaultVersion(); //$NON-NLS-1$
 		try {
 			configureVersions(project, subMonitor.split(1));
-			// add java facet
 			installFacet(project, "jst.java", javaFacet.getVersionString(), null, subMonitor.split(1)); //$NON-NLS-1$
-			project.refreshLocal(IProject.DEPTH_INFINITE, subMonitor.split(1));
```

Comment:
why removing these refreshes?
I remember that before adding them, sometimes facets were not installed correctly because the next facet installation was requiring the previous one to be installed. Calling the refresh was forcing it to have the previous modification taken.

Reply:
if these refreshes are so important then its probably better to add it in the installFacet method.

Reply:
seems better effectively.
can be done with a different task to avoid changing more code in this PR

Reply:
already done and tested. works fine

Reply:
you forgot to modify the number of tick for the monitor in this method